### DISCUSSION TITLE
What will happen to the installed extension in users browsers once HTTPS Everywhere is deprecated?

### DISCUSSION THREAD

**Discussion by Mikaela** [EFForg/https-everywhere#20138] [Q&A] [answered]
There is a discussion at PrivacyGuides on HTTPS Everywhere being deprecated and whether it should be removed from the website (https://github.com/privacyguides/privacyguides.org/issues/145). My opinion is not yet (until the actual deprecation happens, as HTTPS-only mode is so difficult on Edge and Firefox Android currently (https://github.com/privacyguides/privacyguides.org/issues/145#issuecomment-930937863)).

However there was a concern on will people ever remove the deprecated extension from their browsers (https://github.com/privacyguides/privacyguides.org/issues/145#issuecomment-931826984) or will it automatically be removed?

Judging by another discussion/answer, https://github.com/EFForg/https-everywhere/discussions/20137#discussioncomment-1398861, it will become incompatible (practically removing it) at least with Chrome, but how about Firefox?

**Comment by zoracon** [comment, marked as answer]
The extension won't be taken down until late next year (winter, likely). Before then we want to utilize existing mechanisms to remove the extension from Firefox.

There's also a plan to give user prompts, depending on their browser, from the HTTPS Everywhere Home Page. To alleviate the situations where the user has to dig to find a setting. Like in Edge.

Lastly, we will publicly archive the last release of the extension and update the add-on/extensions store pages that it will be no longer updated or maintained before hand.

Any other suggestions are welcome to assist users! We want to make this transition as smooth as we can.
